templates/referent/organizational_chart_macros.html.twig

```diff
@@ -1,39 +1,47 @@
-{% macro orga_chart_row(items, referent, editable) %}
+{% macro orga_chart_row(items, referent, editable, admin) %}
     {% import _self as oc %}
     <ol{% if items is not empty and items|first.level <= 2 %} class="organizational-chart"{% endif %}>
         {% for item in items %}
             <li>
-                {{ oc.orga_chart_item(item, referent, editable) }}
+                {{ oc.orga_chart_item(item, referent, editable, admin) }}
             </li>
         {% endfor %}
     </ol>
 {% endmacro %}
 
-{% macro orga_chart_item(item, referent, editable) %}
+{% macro orga_chart_item(item, referent, editable, admin) %}
     {% import _self as oc %}
     <div>
         <h{{ item.level }}>
             {% if item.referentPersonLinks is defined %}
-                {% if editable %}
-                    <a href="{{ path('app_referent_referent_person_link_edit', {id: item.id}) }}">
+                {% if editable or admin %}
+                    <a href="{{ admin ? admin.generateObjectUrl('edit', item, admin.listFieldDescriptions['batch'].options.route.parameters) : path('app_referent_referent_person_link_edit', {id: item.id}) }}">
                         {% endif %}
-                        {{ item.label }}
+                        <span class="function">
+                            {{ item.label }}
+                        </span>
                         <br />
+                        <span>
                         {% set ref_person_link = get_ref_person_link(item, referent) %}
                         {% if ref_person_link %}
                             {{ ref_person_link.firstName }} {{ ref_person_link.lastName }}
                         {% else %}
                             -
                         {% endif %}
+                    </span>
                         {% if editable %}
```

Comment:
juste if `editable`?

Reply:
@cacahouwete you open `<a>` tag if editable or admin, but you close it only if editable.